python/lsst/afw/geom/linearTransform.py

```diff
@@ -16,6 +16,18 @@ def __reduce__(self):
 def __repr__(self):
     return "LinearTransform(\n%r\n)" % (self.getMatrix(),)
 
+
+def __getitem__(self, k):
+    try:
+        i,j = k
+        if i < 0 or i > 1: raise IndexError
+        if j < 0 or j > 1: raise IndexError
+        return self._getitem_nochecking(i, j)
+    except TypeError:
+        if k < 0 or k > 3: raise IndexError
+        return self._getitem_nochecking(k)
+
```

Comment:
I'm guessing that @pschella was suggesting this as a use case for a 2-d indexing utility function. But @pschella should weigh in.